erp_tutorial_hnn_core.ipynb

```diff
@@ -342,109 +313,91 @@
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "bc11c7c6",
    "metadata": {},
    "outputs": [],
    "source": [
     "# plot layer-specific dipoles\n",
-    "fig, axes = plt.subplots(4, 1, sharex=True, figsize=(6, 6), constrained_layout=True)\n",
+    "fig, axes = plt.subplots(3, 1, sharex=True, figsize=(6, 6), constrained_layout=True, sharey=True)\n",
     "\n",
-    "avg_dpl = average_dipoles(dpls)\n",
     "for idx, layer in enumerate(['agg', 'L2', 'L5']):\n",
-    "    plot_dipole(dpls, ax=axes[idx], layer=layer, show=False)\n",
     "    plot_dipole(avg_dpl, ax=axes[idx], layer=layer, show=False)\n",
-    "    axes[idx].legend(['sim. trial 1', 'sim. trial 2', 'sim. avg.'])\n",
     "\n",
     "# driving input spike histogram\n",
-    "net.cell_response.plot_spikes_hist(ax=axes[3],\n",
-    "                                   spike_types=['evprox', 'evdist']);"
+    "net.cell_response.plot_spikes_hist(spike_types=['evprox', 'evdist']);"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "f74b88ac",
    "metadata": {},
    "source": [
-    "Spiking activity stored in `net.cell_response`:"
+    "We can also plot the spiking activity stored in `net.cell_response`:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "c953ad62",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# plot network spiking activity\n",
     "fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6), constrained_layout=True)\n",
     "net.cell_response.plot_spikes_raster(ax=axes[0], show=False);\n",
     "net.cell_response.plot_spikes_hist(ax=axes[1],\n",
-    "                                   spike_types=['L2_basket', 'L2_pyramidal', 'L5_basket', 'L5_pyramidal'],\n",
+    "                                   spike_types=['L5_pyramidal', 'L2_pyramidal', 'L5_basket', 'L2_basket'],\n",
     "                                   show=False);"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "119e7ad9",
    "metadata": {},
    "source": [
-    "Spectrogram:"
+    "If we are looking at rhythmic activity, we may also be interested in the time-frequency spectrogram of the waveform"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "fc2ae59f",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# plot simulated dipole time-frequency reponse (TFR)\n",
     "fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6),\n",
     "                         constrained_layout=True)\n",
     "\n",
     "exp_dpl.plot(ax=axes[0], show=False)\n",
-    "plot_dipole(dpls, ax=axes[0], layer='agg', show=False)\n",
-    "average_dipoles(dpls).plot(ax=axes[0], show=False)\n",
-    "axes[0].legend(['experimental', 'sim. trial 1', 'sim. trial 2', 'sim. avg.'])\n",
+    "avg_dpl.plot(ax=axes[0], show=False)\n",
+    "axes[0].legend(['experimental', 'simulated avg.'])\n",
```

Comment:
Feel free to disagree but I think it's simpler to just show the average and leave out the single-trial dipoles. Also I wouldn't show the "two different ways of doing the same thing". The `plot_dipole` object exists so we can plot more than one dipole. But then the issue is `simulate_dipole` returns a list with one item. So when providing it to `plot_dipole` one has to do `dpls + exp_dpl`. I thought it's just better to leave this stuff out and have a simple story. Give people the impression that it's all very easy so they feel motivated to try ;-)

Reply:
I actually agree with you here. I was grumbling to myself when I started reading this ;-) but I now think you're right